CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_snapchat_ads_source v0.8.0
+[PR #27](https://github.com/fivetran/dbt_snapchat_ads_source/pull/27) includes the following updates:
+
+## Schema Updates
+
+**2 total changes • 1 breaking change**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| `stg_snapchat_ads__campaign_geo_country_daily_report` | New Staging Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
+| `stg_snapchat_ads__campaign_geo_region_daily_report` | New Staging Model |   |   |  Uses `campaign_geo_region_daily_report` source table  |
+
+Please note that these are disabled by default. To enable them, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+vars:
+    snapchat__using_campaign_country_report: true # Necessary for the stg_snapchat_ads__campaign_geo_country_daily_report model. False by default. Requires the campaign_geo_country_daily_report source table
+    snapchat__using_campaign_region_report: true # Necessary for the stg_snapchat_ads__campaign_geo_region_daily_report model. False by default. Requires the campaign_geo_region_daily_report source table
+```
+
```

Comment:
Could you include an entry on the new passthrough column variables

Reply:
updated